src/modules/rlm_mschap/auth_wbclient.c

```diff
@@ -186,6 +265,7 @@ int do_auth_wbclient(rlm_mschap_t *inst, REQUEST *request,
 
 
 done:
+	fr_connection_release(inst->wb_pool, wb_ctx);
```

Comment:
This can't go here like this - 'done' is sometimes called before wb_ctx is initialised.

Maybe initialise wb_ctx = NULL, then replace this with
```
if (wb_ctx) ...
```